components/contact/ContactForm.jsx

```diff
@@ -1,14 +1,36 @@
 import React, { useState, useEffect, useCallback } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import { toast } from 'react-toastify';
+import { makeStyles } from '@material-ui/core/styles';
 import Grid from '@mui/material/Grid';
 import Button from '@mui/material/Button';
 import TextField from '@mui/material/TextField';
+import InputLabel from '@mui/material/InputLabel';
 import CircularProgress from '@mui/material/CircularProgress';
 import { showFeedbackSuccess, setErrorModal } from '@reducers/ui';
 import { sendGitRequest } from '@reducers/data';
 import 'react-toastify/dist/ReactToastify.css';
 
+const useStyles = makeStyles(theme => ({
+  form: {
+    marginBottom: theme.spacing(20)
+  },
+  formLabel: {
+    fontSize: '18px',
```

Comment:
likely a larger codebase issue and out of scope for this PR but we should prefer avoiding hardcoded font sizes because it has implications for mobile responsiveness

Reply:
I changed this to use the global theme file and set it to the size that matched the figma file